### PR TITLE
Package docs

### PR DESCRIPTION
**Pull request by tjperry07**
* Updated to use html for some headings so linking would work
* Marked where content is missing
* Grammar and formatting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,40 +168,27 @@ This is executed as a single stage:
...
+* [Node Properties](#dynamic-table-dimension-node-properties)
+* [Dynamic Table Options](#dynamic-table-dimension-options)
+* [General Options](#general-dimension-options)
+
+Go to the node and select the **Config tab** to see the Node Properties, Dynamic Table Options and General Options.
   
-![dynamic_table_dimension_options](https://github.com/coalesceio/Dynamic-Table-Nodes/assets/7216836/dab17b38-547b-408a-b864-e27e44728ac3)
+<h3 id="dynamic-table-dimension-node-properties">Node Properties</h3>
+
+[Missing content]
```
Comment (tjperry07) [inline README.md]: @jgontarz This is a empty heading. Not sure what should go here

File: README.md [hunk 2]
```diff
@@ -342,45 +311,30 @@ A table will be dropped if all of these are true:
...
+* [General Options](#dynamic-table-latest-record-general-options)
+
+Go to the node and select the **Config tab** to see the Node Properties, Dynamic Table Options and General Options.
 
-### DAG of Dynamic tables
-When designing DAG of Dynamic tables,it is ideal to specify the target lag as explained in the link below
+<h4 id="dynamic-table-latest-record-node-properties">Dynamic Table Latest Record Node Properties</h4>
 
-[DAG of Dynamic tables](https://docs.snowflake.com/en/user-guide/dynamic-tables-refresh)
+[Missing Content]
```
Comment (tjperry07) [inline README.md]: @jgontarz Missing content in this heading. Not sure what should go here

**Review by prj2001-udn — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -184,40 +168,27 @@ This is executed as a single stage:
...
-## Node Configuration
 The Dynamic Table Dimension has three configuration groups:
 
-* [Node Properties](#node-properties)
-* [Dynamic Table Options](#dynamic-table-options)
-* [Dimension Options](#dimension-options)
+* [Node Properties](#dynamic-table-dimension-node-properties)
+* [Dynamic Table Options](#dynamic-table-dimension-options)
+* [General Options](#general-dimension-options)
+
```
Comment (ghenkhaus) [inline README.md]: This is missing "Dimension Options" config group

**Review by prj2001-udn — COMMENTED** [review]
Changes already commited

File: README.md (hunk 3 above)
Comment (prj2001-udn) [reply README.md]: @ghenkhaus Dimension options added and aligned them accordingly

**Review by prj2001-udn — COMMENTED** [review]
Changes reviewed and updated

**Pull request merged**